src/components/orders/OrderDetails/index.tsx

```diff
@@ -22,7 +41,10 @@ export const OrderDetails: React.FC<Props> = (props) => {
 
   return (
     <Wrapper>
-      <h2>Order details</h2>
+      <h1>
+        Order details
+        {order?.uid && order?.shortId && <TitleUid textToCopy={order.uid} contentsToDisplay={order.shortId} />}
```

Comment:
Nit pick: `uid` and `shortId` are required. If the order exists, they exist too.

So, this simplifies it a little bit

```suggestion
        {order && <TitleUid textToCopy={order.uid} contentsToDisplay={order.shortId} />}
```

Reply:
I'll take the liberty of committing my own suggestion to merge your changes.
Hope you don't mind.